ports/gmp/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "gmp",
   "version-string": "6.2.1",
```

```suggestion
  "version-semver": "6.2.1",
```
This might be [version-semver](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version-semver).